app/full_text_search.py

```diff
@@ -114,47 +98,74 @@ def create_romanized_query_txt(query_txt):
     return enclose_in_parenthesis(romanized_query)
 
 
-def get_person_ids_from_results(romanized_query, results_list):
+def is_query_match(query_txt, values, romanized_values):
+    """
+    To check if the query match the field value
+    Args:
+        query_txt: Search query
+        values: field value for unidecode method
+        romanized_values: romanized field value
+    Returns:
+        Boolean
+    """
+    if not query_txt:
+        return True
```

Comment:
yes, empty location query will match everything.
Inside the search function, it check if the name query is empty or not, if it is empty then it will show error message so this is just for empty location query.